Tools/AP_Bootloader/support.cpp

```diff
@@ -553,11 +553,7 @@ void check_ecc_errors(void)
             break;
         }
 #endif
-        dmaStartMemCopy(dma,
-                        STM32_DMA_CR_PL(0) | STM32_DMA_CR_PSIZE_BYTE |
-                        STM32_DMA_CR_MSIZE_BYTE,
-                        ofs+(uint8_t*)FLASH_BASE, buf, sizeof(buf));
-        dmaWaitCompletion(dma);
+        memcpy((uint8_t*)buf, (uint8_t*)FLASH_BASE+ofs, sizeof(buf));
```

Comment:
this would completely break the functionality as you'd get a hard-fault. this is absolutely the wrong fix